src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.scss

```diff
@@ -117,17 +117,9 @@ $whitecol: white;
   .edit {
     @include btnstyle;
 
-    color: var(--ubs-electric-violet);
-    border-color: var(--ubs-electric-violet);
-    margin-left: 0;
-  }
-
-  .edit:hover {
-    background-color: $whitecol;
-  }
```

Comment:
Hover effect is missing

Reply:
@atsa21 I removed the hover effect because it's absent for similar buttons. For example 'Save changes' on 'ubs-user/profile' page in editing mode.